blues/simulation.py

```diff
@@ -197,12 +197,19 @@ def chooseMove(self):
             correction_factor = (nc_state0['potential_energy'] - md_state0['potential_energy'] + alch_state1['potential_energy'] - nc_state1['potential_energy']) * (-1.0/self.nc_integrator.kT)
             log_ncmc = log_ncmc + correction_factor
 
-        if log_ncmc > randnum:
+        #check if potential energy is greater then precision and reject if that's the case
+        if abs(nc_state1['potential_energy']._value) > 2147483640.:
```

Comment:
Should we try something like this? http://stackoverflow.com/questions/3018758/determine-precision-and-scale-of-particular-number-in-python